gated-infrastructure-deploy/gated-infrastructure-deploy.yml

```diff
@@ -21,10 +21,13 @@
     type: object
   - name: TerraformOutputVariables
     type: object
+  - name: terraformazurecontainer
+    value: ukhydrographicoffice/terraform-azure:1.3.9
```

Comment:
We don't want the specify the value here, change this line to be "type: string". The calling pipeline will provide the value.